test/test_nn.py

```diff
@@ -7062,7 +7061,7 @@ class TestFusionEval(TestCase):
     @given(X=hu.tensor(shapes=((5, 3, 5, 5),), dtype=np.double),
            running_mean=hu.tensor(shapes=(6,), dtype=np.double),
            running_var=hu.tensor(shapes=(6,), dtype=np.double))
-    def test_fuse_module_eval_numerics(self, X, running_mean, running_var):
+    def test_fuse_module_conv_eval_numerics(self, X, running_mean, running_var):
```

Comment:
When I used `@parametrize_test("affine", [True, False])` and add `affine` as an argument, the test wouldn't run. I have no idea why. The error

```
======================================================================
ERROR: test_fuse_module_conv_eval_numerics (__main__.TestFusionEval)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/thien/miniforge3/envs/tv_test/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 2652, in wrapper
    method(*args, **kwargs)
  File "/Users/thien/miniforge3/envs/tv_test/lib/python3.10/contextlib.py", line 79, in inner
    return func(*args, **kwds)
TypeError: accept.<locals>.test_fuse_module_conv_eval_numerics() missing 1 required positional argument: 'affine'
```